lib/l10n/app_en.arb

```diff
@@ -779,5 +787,9 @@
   "repairDownloads": "Repair issues with download files or metadata.",
   "@repairDownloads": {
     "description": "Tooltip for downloads repair button."
+  },
+  "requireWifiForDownloads": "Require wifi when Downloading.",
+  "@requireWifiForDownloads": {
+    "description": "Description of setting for requireWifiForDownloads."
```

Comment:
The description is geared towards translators on Weblate, so it shouldn't be technical, but instead describe the context where the string will be used. So for example `Label for the setting that toggles between downloading only over WiFi and downloading over any kind of internet connection. If active, Finamp will only download over WiFi.`

(It doesn't have to be that long, just try not to reference code or the keys for the translation tokens, at least not exclusively :) )